modular_skyrat/modules/modular_weapons/code/energy.dm

```diff
@@ -248,3 +362,22 @@
 	projectile_type = /obj/projectile/beam/disabler/bounce
 
 
+/obj/item/ammo_casing/caseless/laser_smg/lethal
+	name = "type V1 energy projectile"
+	desc = "An energy cell that once triggered, creates a lethal projectile, melting it's own casing in the process."
+	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/ammo.dmi'
+	icon_state = "plasma_shell"
+	worn_icon_state = "shell"
+	caliber = "energy blast"
+	custom_materials = list(/datum/material/iron=4000,/datum/material/plasma=250)
+	projectile_type = /obj/projectile/beam/laser/single/laser_smg
+
+/obj/item/ammo_casing/caseless/laser_smg/nonlethal
+	name = "type V2 energy projectile"
+	desc = "An energy cell that once triggered, creates a non-lethal projectile, melting it's own casing in the process."
+	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/ammo.dmi'
+	icon_state = "disabler_shell"
+	worn_icon_state = "shell"
+	caliber = "electrode"
+	custom_materials = list(/datum/material/iron=4000,/datum/material/plasma=250)
+	projectile_type = /obj/projectile/energy/electrode/laser_smg
```

Comment:
You could use inheritance to reduce a lot of the code needed here. Please send me a message on discord if you need help with this.

Reply:
I don't have your discord, so if you'll send it to me, then sure.